src/lib/consensus/proof_of_stake.ml

```diff
@@ -3256,10 +3304,18 @@ let%test_module "Proof of stake tests" =
           exists Coda_base.Frozen_ledger_hash.typ
             ~compute:(As_prover.return snarked_ledger_hash)
         in
+        let%bind constants_checked =
```

Comment:
These should not be `existed` in, because then they can be anything. You should get them from the previous consensus state.

Reply:
Oh -- this is a test so I guess it's fine